resources/views/components/game/top-achievers/score-row.blade.php

```diff
@@ -15,5 +16,12 @@
             <x-user.avatar display="icon" icon-size="xs" :user="$user" />
             <x-user.avatar hasHref="true" :user="$user" />
         </td>
-        <td class="text-right">{{ localized_number($score) }}</td>
+        <td>
+            <div class="flex items-center gap-1 float-right">
+                @if ($score === $maxScore)
+                    <div class="rounded-full bg-[gold] light:bg-yellow-600 w-2 h-2"></div>
```

Comment:
Recommend adding a `title="Mastered"` attribute.

In other areas, there's contextual information explaining what the dot means (either a label or a legend elsewhere). For accessibility, we probably need something here too.